test/sequential/test-benchmark-child-process.js

```diff
@@ -8,13 +8,16 @@ const path = require('path');
 
 const runjs = path.join(__dirname, '..', '..', 'benchmark', 'run.js');
 
+const env = Object.assign({}, process.env,
+                          { NODEJS_BENCHMARK_ZERO_ALLOWED: 1 });
+
 const child = fork(runjs, ['--set', 'dur=0',
```

Comment:
Not really related to this PR, but if @BethGriggs wants to do that that's fine too.